include/pistache/endpoint.h

```diff
@@ -19,6 +19,7 @@ class Endpoint {
         friend class Endpoint;
 
         Options& threads(int val);
+        Options& threadsName(std::string val);
```

Comment:
const std::string& (pass by ref, not pass by value, since we are not using X-value/move semantics).